### DISCUSSION TITLE
[Optionally?] Move from draft when adding reviewers

### DISCUSSION THREAD

**Discussion by brianjmurrell** [community/community#37637] [Pull Requests]
I often forget to move a PR from draft to ready-for-review when I'm ready and have added reviewers.

It would be nice/neat to ask if a PR should be changed to ready-for-review state when the user starts adding reviewers.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @brianjmurrell ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
